src/shared/utils/tooltip.ts

```diff
@@ -163,18 +166,22 @@ export const showTooltipAtCoordinates = (
   document.body.addEventListener('scroll', hideTooltip, true);
   document.body.addEventListener('wheel', hideTooltip, true);
 
-  function hideTooltip() {
-    tooltip?.remove();
-    cleanup?.();
-    document.body.removeEventListener('click', onClick, true);
-    document.body.removeEventListener('scroll', hideTooltip, true);
-    document.body.removeEventListener('wheel', hideTooltip, true);
+  function hideTooltip(e?: Event) {
+    const target = e?.target as Node;
```

Comment:
```
const target = e?.target as Node | null;
```
This doesn't really change as you're already checking for `target`'s existence later. Suggesting just for consistency.